src/Traits/LucidTrait.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Vinelab\Tracing\Traits;
+
+use Illuminate\Log\Events\MessageLogged;
+use Illuminate\Support\Facades\Event;
+use Lucid\Foundation\Events\FeatureStarted;
+use Lucid\Foundation\Events\JobStarted;
+use Lucid\Foundation\Events\OperationStarted;
+use Trellis\Clients\Facades\Intercept;
+use Vinelab\Tracing\Facades\Trace;
+use Vinelab\Tracing\Propagation\Formats;
+
+/**
+ * Provides tracing configuration for Lucid based projects
+ */
+trait LucidTrait
```

Comment:
The name is a bit too generic. Recommended to use verbs for traits and omit the word Trait in accordance with Laravel conventions (e.g. _TraceLucidArchitecture_).